### DISCUSSION TITLE
intersectsWithObject between objects in and not in activeSelection

### DISCUSSION THREAD

**Discussion by chau-a-nguyen** [fabricjs/fabric.js#9017] [Q&A] [answered]
I am using fabricJS 4.3.1.

When an object is part of an active selection of multiple objects, the this.intesectsWithObject no longer returns true for objects it is "intersecting" with that are not part of the selection.

How to detected if it intersects with object regardless of the activeSelection (as if they were all at the root level? Thanks!

**Comment by ShaMan123** [comment, marked as answer]
#7670
